app/helpers/article_helper.rb

```diff
@@ -53,6 +53,17 @@ def mark_html_safe(options = {})
     options[:value].map(&:to_s).to_sentence(separators).html_safe
   end
 
+  def sanitize_fulltext(options = {})
+    return unless options[:value].present?
+    separators = options.dig(:config, :separator_options) || {}
+    white_list_sanitizer = Rails::Html::WhiteListSanitizer.new
+    textblock = options[:value].map(&:to_s).to_sentence(separators)
+    textblock = Nokogiri::HTML.fragment(CGI.unescapeHTML(textblock))
+    textblock.search('anid').remove
+    textblock = textblock.to_html
+    white_list_sanitizer.sanitize(textblock).html_safe
```

Comment:
i think this can just be `sanitize` (without the `Rails::Html::WhiteListSanitizer.new`) as I think that's the default in Rails 5.